### DISCUSSION TITLE
Bibliotecas de Ícones

### DISCUSSION THREAD

**Discussion by onicolasdelfino** [oeduardostein/oeduardostein#7] [Q&A] [answered]
Alguém poderia me ajudar indicando algumas bibliotecas de ícones?

**Comment by oeduardostein** [comment, marked as answer]
Claro! Aqui está uma lista organizada de bibliotecas de ícones:

Bibliotecas de Ícones
Font Awesome
Descrição: Uma das bibliotecas mais populares, com uma vasta coleção de ícones vetoriais.
Link: [Font Awesome](https://fontawesome.com/)

Material Icons
Descrição: Ícones baseados no Material Design do Google, ideais para aplicações web e móveis.
Link: Material Icons

Ionicons
Descrição: Focada em aplicações móveis e web, com design limpo e moderno.
Link: [Ionicons](https://ionicons.com/)

Feather Icons
Descrição: Ícones simples e bonitos, projetados para serem minimalistas e escaláveis.
Link: [Feather Icons](https://feathericons.com/)

Bootstrap Icons
Descrição: Criada para uso com o framework Bootstrap, com ícones que se integram bem a projetos Bootstrap.
Link: Bootstrap Icons

Heroicons
Descrição: Ícones bonitos e cuidadosamente projetados, ideais para interfaces de usuário.
Link: [Heroicons](https://heroicons.com/)

Boxicons
Descrição: Biblioteca de ícones simples e flexíveis, com um conjunto variado de opções.
Link: [Boxicons](https://boxicons.com/)

Lucide
Descrição: Biblioteca de ícones de código aberto, moderna e reimaginada a partir dos ícones do Feather.
Link: [Lucide](https://lucide.dev/)

Essas bibliotecas oferecem uma ampla gama de ícones para atender diferentes necessidades em seus projetos.

**Comment by onicolasdelfino** [comment]
Muito Obrigado!

**Comment by oeduardostein** [comment]
De nada!
